source/MaterialXGenShader/ShaderGraph.cpp

```diff
@@ -675,7 +675,7 @@ ShaderGraphPtr ShaderGraph::create(const ShaderGraph* parent, const string& name
         {
             // A free floating output.
             outputParent = output->getConnectedNode();
-            interface = outputParent->asA<InterfaceElement>();
+            interface = outputParent ? outputParent->asA<InterfaceElement>() : nullptr;
```

Comment:
outputParent is accessed several times above, so this nullptr check as no effect.

Reply:
I'm not sure the exact situation but I was getting null back from this for outputParent sometimes.

Reply:
That might happen if `outputParent->asA<InterfaceElement>()` returns null, if it's not always an InterfaceElement so the dynamic cast will fail.